modules/javafx.graphics/src/main/java/com/sun/javafx/font/PrismFontFactory.java

```diff
@@ -238,6 +238,13 @@ private PrismFontFile createFontResource(String name,
                                              boolean register, boolean embedded,
                                              boolean copy, boolean tracked) {
         String key = (filename+index).toLowerCase();
+        /*
+         * macOS: we need to load unique fonts for regular and bold.
+         * Probably this should be handled elsewhere
+         */
+        if (isMacOSX && name.startsWith("System ")) {
```

Comment:
`name` can be null, so you need a null check here. This is the cause of the unit test failures.

Reply:
fixed